data/build.gradle.kts

```diff
@@ -84,6 +84,9 @@ kotlin {
             languageSettings.apply {
                 optIn("kotlinx.coroutines.ExperimentalCoroutinesApi")
             }
+            dependencies {
+                implementation("com.google.truth:truth:1.0.1")
+            }
```

Comment:
I don't think we need another assertion library.

Reply:
> I don't think we need another assertion library.

Well, the Truth library makes assertions more readable and easy to understand unlike the normal assertions used throughout the hole tests (from what I have seen). 
Perhaps we can replace the project assertions library by this one. Read a bit about it, you might change your mind 😀

Reply:
I know about it :sweat_smile:. I just don't think we need it. Tests can be made more readable without it. Please remove it.